### PR TITLE
Add new path in candlepin_error_log spec file

### PR DESCRIPTION
**Pull request by sagaraivale**
Add new path in candlepin_error_log spec file.

### CODE REVIEW

**Comment by sagaraivale** [comment]
@vishwanathjadhav @xiangce  
 Kindly review it...!!

File: insights/config/specs.py [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "brctl_show"                : CommandSpec("/usr/sbin/brctl show"),
     "candlepin.log"             : First([SimpleFileSpec(r"var/log/candlepin/candlepin.log", large_content=True),
                                     SimpleFileSpec(r"sos_commands/foreman/foreman-debug/var/log/candlepin/candlepin.log", large_content=True)]),
-    "candlepin_error_log"       : NoneGroup([SimpleFileSpec(r"sos_commands/foreman/foreman-debug/var/log/candlepin/error.log", large_content=True)]),
+    "candlepin_error_log"       : First([SimpleFileSpec(r"var/log/candlepin/error.log", large_content=True),
+                                    SimpleFileSpec(r"sos_commands/foreman/foreman-debug/var/log/candlepin/error.log", large_content=True)]),
```
Comment (bfahr) [inline insights/config/specs.py]: @sagaraivale I believe these specs should still be wrapped in a `NoneGroup` since they are targeted for archives only.

Comment (bfahr) [reply insights/config/specs.py]: @xiangce thanks for clearing that up.

**Comment by xiangce** [comment]
@sagaraivale - It's ready to merge if you re-submit this PR to the master branch, thanks.

**Pull request merged**